content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -109,35 +109,51 @@ To set up a Uruware account, first be sure to have a valid Odoo subscription. Th
 proceeding steps.
 
 Locate the Uruware credentials settings by navigating to the :menuselection:`Accounting -->
-Configuration --> Settings`. Scroll down to the :guilabel:`Uruguayan Localization` section and click
-on :guilabel:`Create Uruware Account`.
+Configuration --> Settings`. Scroll down to the :guilabel:`Uruguayan Localization` section, select
+the environment (:guilabel:`Production` or :guilabel:`Testing`), and click on :guilabel:`Create
+Uruware Account`.
 
-Upon doing so, an email is sent to the address associated with your Odoo subscription with the
-username and password to enter Uruware's portal and set up your account.
+Upon doing so, an email is sent within 48hrs to the address associated with your Odoo subscription
+with the password to enter Uruware's portal and set up your account.
 
 .. tip::
-   Your :guilabel:`Tax ID` needs to be setup to be able to create your account.
+  - The company's :guilabel:`Tax ID` needs to be set up to be able to create an Uruware account. The email with the credentials is not immediate; it might take up to 48 hours for the account to be created.
+  - The password sent expires after 24 hours. In this case, reset it by using the *Forgot Password* link in Uruware's portal.
 
 .. note::
    This action will create an account with Uruware with the following information:
 
    - Legal name (razón social)
    - RUT from the company
-   - User name (RUT.odoo)
+   - Username (the Odoo subscription email or `RUT`.odoo. For example: `213344556677.odoo`)
    - Odoo database link
 
    To ensure your account is created correctly, please add any missing information from above.
 
-Once the account is created, a confirmation email is sent to the email linked to your subscription
-in Odoo, containing the credentials to configure the account directly in the Uruware
-`testing portal <https://odootest.ucfe.com.uy/Gestion/>`_ or
+Once the account is created and you have received the email containing the credentials, configure
+your accounts directly in the Uruware `testing portal <https://odootest.ucfe.com.uy/Gestion/>`_ or 
 `production portal <https://prod6109.ucfe.com.uy/Gestion/>`_:
 
-Use the account credentials in the email to log in to the portal.
+Use the account credentials in the email to log in to the to the corresponding (`test
+<https://odootest.ucfe.com.uy/Gestion/>`_ or `production <https://prod6109.ucfe.com.uy/Gestion/>`_)
+portal.
+
+- Complete and correct the company's information
+- Add your digital certificate
+- Add your :abbr:`CAEs (Constancia de Autorización para Emisión)` for each document-type you plan to 
+issue.
+- Configure the format of the PDF to be printed and sent to your customers
```

Comment:
```suggestion
In Uruware's portal, the following steps are needed to be able to issue invoices from Odoo:
1. Complete and correct the company's information.
2. Add your digital certificate.
3. Add your :abbr:`CAEs (Constancia de Autorización para Emisión)` for each document-type you plan to 
issue.
4. Configure the format of the PDF to be printed and sent to your customers.
```
@samueljlieber this is my suggestion, but feel free to correct it if you think it is necessary. Thanks!

Reply:
@fvz-odoo looks good! Only changes are to add a blank line before the numbered list and to use dynamic numbering syntax :)
```suggestion
In Uruware's portal, the following steps are needed to be able to issue invoices from Odoo:

#. Complete and correct the company's information.
#. Add your digital certificate.
#. Add your :abbr:`CAEs (Constancia de Autorización para Emisión)` for each document-type you plan to 
issue.
#. Configure the format of the PDF to be printed and sent to your customers.
```